### DISCUSSION TITLE
Pendo SDK support for SwiftUI app

### DISCUSSION THREAD

**Discussion by coder-selvarajan** [pendo-io/pendo-mobile-sdk#145] [Q&A] [answered]
Hello, I need some help with codeless event collection in my SwiftUI app. My app has several button controls. I've integrated the SDK into my app and am reviewing the Raw Events dashboard. I see 'RAClick' events from my buttons, but I cannot identify which particular button is being clicked. I can see that the accessibilityLabel and accessibilityIdentifier have some IDs, but not the actual values. Is there a way to extract those identifiers? Or is there a limitation in Pendo SDK support for SwiftUI apps?

**Comment by MikePendo** [comment, marked as answer]
Hi 
The reason why u dont see the exact identifiers/labels/text/hints is coz they are hashed for your end user.
If you want to see the non hashed values please capture the page and select the feature, under the feature rules u will see the data we manage to collect. on click event the "user data" like identifiers/labels/text/hints is hashed to prevent any PII issue. 
Eventually we compare the hashed values with the hashed values we captured when u did the pairing capture page.
Please let us know if you have any other questions
